src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="/"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="label">
+        Surname:
+          <input
+          type="text"
+          name="Surname"
+          pattern="\w+"
+          autocomplete="off"
+          >
```

Comment:
please stick with indentations as per [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md#:~:text=If%20the%20HTML%2Delement%20has%20long%20attribute)
```suggestion
          Surname:
          <input
            type="text"
            name="Surname"
            pattern="\w+"
            autocomplete="off"
          >
```